build.sbt

```diff
@@ -1,13 +1,14 @@
 import ReleaseTransformations._
+import sbtcrossproject.CrossPlugin.autoImport.{crossProject, CrossType}
 
 lazy val intervalsetSettings = Seq(
   organization := "com.rklaehn",
   scalaVersion := "2.12.1",
-  crossScalaVersions := Seq("2.11.11", "2.12.1"),
+  crossScalaVersions := Seq("2.12.1"),
```

Comment:
I've removed support for Scala 2.11 since `sonicreducer` version 0.5 wasn't published for that Scala version. Or at least the jar is no longer available for download.